packages/ses/package.json

```
@@ -73,7 +73,9 @@
     "prepare": "npm run clean && npm run build",
     "qt": "ava",
     "test": "tsd && ava",
-    "test:platform-compatibility": "node test/package/test.cjs"
+    "test:platform-compatibility": "node test/package/test.cjs",
+    "test:create-hermes-bin-symlinks": "./scripts/hermes-bin-symlinks.sh",
+    "test:hermes": "./scripts/hermes.sh"
```

```suggestion
    "test:hermes": "yarn build && ./scripts/hermes.sh"
```

since it's too easy/annoying to forget to build locally first after each/any change to SES, to get immediate feedback from Hermes

---

or keep CI consistent with `yarn build` there for each job

update `hermes.sh` instead to check if built already, if not, build